client/layout/popup-search/index.js

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/interactive-supports-focus */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize, useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import HelpSearchCard from 'calypso/blocks/inline-help/inline-help-search-card';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { getInlineHelpAdminSectionSearchResultsForQuery } from 'calypso/state/inline-help/selectors/get-inline-help-search-results-for-query';
+import hasInlineHelpAPIResults from 'calypso/state/selectors/has-inline-help-api-results';
+import { selectResult } from 'calypso/state/inline-help/actions';
+import { localizeUrl } from 'calypso/lib/i18n-utils';
+import QueryInlineHelpSearch from 'calypso/components/data/query-inline-help-search';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import getSearchQuery from 'calypso/state/inline-help/selectors/get-search-query';
+
+export function PopUpSearch( props ) {
+	const translate = useTranslate();
+
+	const onChildClick = ( e ) => e.stopPropagation();
+
+	const onResultClick = ( link ) => {
+		props.onClose();
+		window.location.href = localizeUrl( link );
```

Comment:
This won't carry the query parameter, which means the action search will be gone once we navigate away from here if someone enables this feature through the `flags` query parameter. It'd be a negative experience for the internal test, so let's find a better way :)

Reply:
Nice catch! I will fix this.